culture/remote/README.md

```diff
@@ -0,0 +1,43 @@
+# Working with our Remote Teammates
+
+At Sparkbox, we have the opportunity to work with many different people in many different locations. Whether its' with our co-workers or our clients, sometimes, we all have to work remote.
+
+## Tips for Non-remote folks When Working With Remote Colleagues
+
+Let's be more inclusive of our remote coworkers and clients. Here are some ways how.
+
+- When a conversation becomes more than a quick back-and-forth on [Slack][], spin up a [Zoom][] and encourage all parties to join.
+- Join meetings 2-5 minutes early, to encourage friendly, human banter. Avoid jumping directly into tasking whenever possible.
+- Use a headset/headphone in public spaces. Avoid noisy or high-traffic areas, even if they don't seem loud to you.
+- Let people know if you're not using a headset, or if they are "on blast" in public spaces.
+- Activities with physical paper, whiteboards, or other items should be avoided. Use online tools such as [draw.io][] instead.
+- When one person is remote on a meeting, everyone should be remote. Find individual spaces, such as your desk, so that everyone uses headphones and is on the call separately. This equalizes the speaking floor so that everyone involved has the same opportunity to share and listen. No person has to compete with a large conference room where the video screen is quickly forgotten.
+- If you _are_ in a situation with a big group in one room and a few remote people on a call, designate a "point person" in the large group space as an advocate for your remote friends.
+- Make your video meetings more friendly:
+  - Set default Zoom settings to join without sound.
+  - Use "gallery view" instead of "current speaker."
+- Try not to ask for things from your remote workers during lunch or outside normal working hours. "[Remote Guilt][]" is a real thing, and people who are remote have no way to tell that the Dayton office is pretty shut down around 4:30pm.
+
+## Some of Our Favorite Resources
+
+Here are some links to articles and videos we love.
+
+- [Working on Distributed Teams at Mozilla][] (video) - Mozilla
```

Comment:
This link is taking me to the YouTube landing page instead of the intended video

Reply:
😭